src/lib/buyables.ts

```diff
@@ -34,6 +34,30 @@ const Buyables: Buyable[] = [
 		},
 		qpRequired: 35,
 		gpCost: 10_000
+	},
+	{
+		name: 'Barrows gloves',
```

Comment:
what about the other gloves? people will want to collect, and have t hem all in CL